src/table/components/withSelections.tsx

```diff
@@ -0,0 +1,26 @@
+import React from 'react';
+import { useContextSelector, TableContext } from '../context';
+import { getSelectionStyle } from '../utils/styling-utils';
+import { getCellSelectionState, SelectionActions, TSelectionActions } from '../utils/selections-utils';
+import { hocProps, SelectionState } from '../../types';
+
+export default function withSelections(CellComponent: (props: hocProps) => JSX.Element) {
+  const HOC = (props: hocProps) => {
+    const { cell, styling, announce } = props;
+    const cellSelectionState = useContextSelector(TableContext, (value) =>
```

Comment:
It is strange that `const cellSelectionState = useContextSelector(TableContext, (value) => getCellSelectionState(cell, value));` worked before.

Reply:
Never mind, now I see.